ejb/WFLY-6892-ejb-access-logging.adoc

```diff
@@ -0,0 +1,387 @@
+                <xs:element name="formatter" type="accessLogFormatterType" minOccurs="0"/>
+                <xs:element name="file" type="pathType" minOccurs="1"/>
+                <xs:element name="append" type="booleanValueType" minOccurs="0"/>
+            </xs:all>
+            <xs:attribute name="autoflush" type="xs:boolean" use="optional" default="true"/>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true"/>
+        </xs:complexType>
+
+        <xs:complexType name="periodicFileAccessLogType">
+            <xs:annotation>
+                <xs:documentation>
+                    Defines an access log which writes to a file, rotating the log after a time period derived from the given
+                    suffix string, which should be in a format understood by java.text.SimpleDateFormat.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:all>
+                <xs:element name="encoding" type="valueType" minOccurs="0"/>
+                <xs:element name="formatter" type="accessLogFormatterType" minOccurs="0"/>
+                <xs:element name="file" type="pathType"/>
+                <xs:element name="suffix" type="valueType"/>
+                <xs:element name="append" type="booleanValueType" minOccurs="0"/>
+            </xs:all>
+            <xs:attribute name="autoflush" type="xs:boolean" use="optional" default="true"/>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true"/>
+        </xs:complexType>
+
+        <xs:complexType name="sizeFileAccessLogType">
+            <xs:annotation>
+                <xs:documentation>
+                    Defines an access log which writes to a file, rotating the log after the size of the file grows beyond a
+                    certain point and keeping a fixed number of backups.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:all>
+                <xs:element name="encoding" type="valueType" minOccurs="0"/>
+                <xs:element name="formatter" type="accessLogFormatterType" minOccurs="0"/>
+                <xs:element name="file" type="pathType"/>
+                <xs:element name="rotate-size" type="sizeType" minOccurs="0"/>
+                <xs:element name="max-backup-index" type="positiveIntType" minOccurs="0"/>
+                <xs:element name="suffix" type="valueType" minOccurs="0"/>
+                <xs:element name="append" type="booleanValueType" minOccurs="0"/>
+            </xs:all>
+            <xs:attribute name="autoflush" type="xs:boolean" use="optional" default="true"/>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true"/>
```

Comment:
Same comment as above for this.

Reply:
Makes sense, and I'll remove this attribute (enabled).